tests/common.go

```diff
@@ -995,7 +995,13 @@ func ValidateVolumes(ctx *scheduler.Context, errChan ...*chan error) {
 				log.Infof("Using vol scale factor of %d for app %s", volScaleFactor, ctx.App.Key)
 			}
 			scaleFactor := time.Duration(Inst().GlobalScaleFactor * volScaleFactor)
-			err = Inst().S.ValidateVolumes(ctx, scaleFactor*defaultVolScaleTimeout, defaultRetryInterval, nil)
+			volumeTimeout := scaleFactor * defaultVolScaleTimeout
+			// If provisioner is IBM increase the timeout to 8 min
+			if Inst().Provisioner == "ibm" {
+				volumeTimeout = scaleFactor * 8 * time.Minute
```

Comment:
Instead of hard-coding this to `8 * time.Minute` can we keep this in a const called `IbmVolScaleTimeout` ?
If we feel the 8 minutes is not sufficient in future then it is easy to change.

Reply:
Done